server/sonar-server/src/main/java/org/sonar/ce/queue/CeQueueImpl.java

```diff
@@ -70,61 +75,122 @@ public CeQueueImpl(DbClient dbClient, UuidFactory uuidFactory, DefaultOrganizati
 
   @Override
   public CeTask submit(CeTaskSubmit submission) {
-    checkState(!submitPaused.get(), "Compute Engine does not currently accept new tasks");
+    return submit(submission, EnumSet.noneOf(SubmitOption.class)).get();
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Call "Optional#isPresent()" before accessing the value. ([squid:S3655](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3655))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-261%2Fnear-real-time-views&issues=AWH8OoBRa5Ltb7kAT0bQ&open=AWH8OoBRa5Ltb7kAT0bQ)

[ISSUE_KEY]: AWH8OoBRa5Ltb7kAT0bQ